@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -2,11 +2,22 @@
   <template :v-if="computedItems.length">
     <slick ref="slick" :options="computedSlickOptions">
       <div v-for="item in computedItems" :key="item.id">
-        <a :href="item.destinationUrl">
+        <div v-if="computedSlickOptions.slidesToShow === 1" style="position: relative;">
+          <a :href="item.destinationUrl">
           <img v-if="item.imageUrl" :src="item.imageUrl" :alt="item.altText">
           <template v-if="!item.imageUrl">{{ item.title }}</template>
+          <div v-if="item.altText" style="width: 100%; position: absolute; bottom: 0; background: rgb(0,0,0,0.5); color: white;">
+            <p style="margin: 0; padding: 10px;" v-bind:style="alignText">{{ item.altText }}</p>
+          </div>
         </a>
-        <p v-if="!item.imageUrl">{{ item.description }}</p>
+        </div>
+        <div v-if="computedSlickOptions.slidesToShow > 1" style="padding: 10px;">
+          <p v-if="!item.imageUrl" style="min-height: 7rem;">{{ item.description }}</p>
+          <a :href="item.destinationUrl">
+            <img v-if="item.imageUrl" :src="item.imageUrl" :alt="item.altText">
+            <template>{{ item.title }}</template>
```

Comment:
Does a `<template>` inside a `<template>` render?

Reply:
Not sure I follow. This code doesn't show `<template>` inside a `<template>`?

Reply:
line 18 `<template>` is inside line 2 `<template>`, IIRC, vue only renders on `<template>` at a time.

Reply:
More vue magic :tophat: :rabbit: :sparkles: